Demo/Demo/ViewModels/BaseViewModel.swift

```diff
@@ -184,11 +190,20 @@ class BaseViewModel: ObservableObject, PayPalWebCheckoutDelegate, CardDelegate {
             let authorizeResult = try? await DemoMerchantAPI.sharedService.authorizeOrder(
                 orderID: result.orderID, selectedMerchantIntegration: selectedMerchantIntegration
             )
-            let status = authorizeResult?.status ?? ""
-            updateTitle("Order ID:\(result.orderID) status: \(status)")
+            displayResult(cardResult: result, orderResult: authorizeResult)
         }
     }
     
+    func displayResult(cardResult: CardResult, orderResult: Order?) {
```

Comment:
since the first property in the method signature is result I think we can do:
```suggestion
    func displayResult(_ cardResult: CardResult, orderResult: Order?) {
    // OR
    func display(_ cardResult: CardResult, orderResult: Order?) {
```